docs/hub/model-card-annotated.md

```diff
@@ -0,0 +1,308 @@
+
+* The **developer** is necessary for filling out [Training Procedure](#training-procedure) and [Technical Specifications](#technical-specifications). They are also particularly useful for the “Limitations” section of [Bias, Risks, and Limitations](#bias-risks-andlimitations). They are responsible for providing [Results](#results) for the Evaluation, and ideally work with the other roles to define the rest of the Evaluation: [Testing Data, Factors & Metrics](#testing-data-factors-and-metrics).
+
+* The **sociotechnic** is necessary for filling out “Bias” and “Risks” within [Bias, Risks, and Limitations](#bias-risks-and-limitations), and particularly useful for “Out of Scope Use” within [Uses](#uses).
+
+* The **project organizer** is necessary for filling out [Model Details](#model-details) and [Uses](#uses). They might also fill out
+[Training Data](#training-data). Project organizers could also be in charge of [Citation](#citation), [Glossary](#glossary), 
+[Model Card Contact](#model-card-contact), [Model Card Authors](#model-card-authors), and [More Information](#more-information).
+
+_Instructions are provided below, in italics._
+
+Template variable names appear in `monospace`.
+
+# Model Name
+
+**Section Overview:**  Provide the model name and a 1-2 sentence summary of what the model is. 
+
+`model_id`
+
+`model_summary`
+
+# Table of Contents
+
+**Section Overview:** Provide this with links to each section, to enable people to easily jump around/use the file in other locations with the preserved TOC/print out the content/etc.
+
+# Model Details
+
+**Section Overview:** This section provides basic information about what the model is, its current status, and where it came from. It should be useful for anyone who wants to reference the model.
+
+
+## Model Description
+
+
+`model_description`
+
+_Provide basic details about the model. This includes the architecture, version, if it was introduced in a paper, if an original implementation is available, and the creators. Any copyright should be attributed here. General information about training procedures, parameters, and important disclaimers can also be mentioned in this section._
+
+
+* **Developed by:** `developers`
+
+_List (and ideally link to) the people who built the model._
+
+
+
+* **Shared by [optional]:** `shared_by`
+
+_List (and ideally link to) the people/organization making the model available online._
+
+
+
+* **Model type:** `model_type`
+
+_You can name the “type” as:_
+
+_1. Supervision/Learning Method_
+
+_2. Machine Learning Type_
+
+_3. Modality_
+
+* **Language(s)** [NLP]: `language`
+
+_Use this field when the system is a natural language processing system._
```

Comment:
Could potentially be for `text-to-image` and ML for audio (e.g. text to speech) too

Reply:
That's a great point; NLP here is being used more as a descriptor term rather than a field term, but that's indistinguishable! I can change to `...when the system uses or processes natural (human) language`.